vendors/renesas/boards/rx65n-rsk/aws_tests/config_files/FreeRTOSIPConfig.h

```diff
@@ -324,4 +317,7 @@ void vApplicationMQTTGetKeys( const char ** ppcRootCA,
                               const char ** ppcClientCert,
                               const char ** ppcClientPrivateKey );
 
+/* Let NetworkInterface check the free space in the working queue for the IP-task. */
+#define ipconfigCHECK_IP_QUEUE_SPACE			1
```

Comment:
I'd like to see the config for aws_tests and for aws_demos be the same unless there is a compelling reason to have differences.  Are the changes I see to the demo config above already in this file?

Reply:
In the existing repo, the are several differences between the two IP-config files.
After PR #1533, there will be 2 differences left:

~~~c
 /* Each TCP socket has a circular buffers for Rx and Tx, which have a fixed
  * maximum size.  Define the size of Rx buffer for TCP sockets. */
-#define ipconfigTCP_RX_BUFFER_LENGTH                   ( 1460 * 8 )
+#define ipconfigTCP_RX_BUFFER_LENGTH                   ( 1460 * 3 )
 
 /* Define the size of Tx buffer for TCP sockets. */
-#define ipconfigTCP_TX_BUFFER_LENGTH                   ( 1460 * 8 )
+#define ipconfigTCP_TX_BUFFER_LENGTH                   ( 1460 * 3 )
~~~

I don't know why the testing version needs larger TCP stream buffers. I have no problem if aws_demo gets the same configuration as aws_test.
Although I do not understand a difference in clock speeds:

~~~c
-#define configCPU_CLOCK_HZ				( 120000000UL )
+#define configCPU_CLOCK_HZ				( 96000000UL )
-#define configPERIPHERAL_CLOCK_HZ		( 60000000UL )
+#define configPERIPHERAL_CLOCK_HZ		( 48000000UL )
~~~
Found in:
  vendors\renesas\boards\rx65n-rsk\aws_tests\config_files\FreeRTOSConfig.h
  vendors\renesas\boards\rx65n-rsk\aws_demos\config_files\FreeRTOSConfig.h